### PR TITLE
Adding Login template

### PR DESCRIPTION
**Pull request by wacevedo**

### CODE REVIEW

File: src/Template/Users/login.ctp [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
+                <span class="fa fa-user form-control-feedback"></span>
+            </div>
+            <div class="form-group has-feedback">
+                <?= $this->Form->input('password', ['type'=>'password', 'class'=>'form-control', 'placeholder'=>'Password', 'label' => false]) ?>
+            <span class="fa fa-lock form-control-feedback"></span>
+          </div>
+        </fieldset>
+
+        <div class="row">
+        <div class="col-xs-8">
```
Comment (csuriel) [inline src/Template/Users/login.ctp]: Please add the correct indentation for HTML tags in cascade.

File: src/Template/Users/login.ctp [hunk 2]
```diff
@@ -1,11 +1,44 @@
...
-        <legend><?= __('Please enter your username and password') ?></legend>
-        <?= $this->Form->input('username') ?>
-        <?= $this->Form->input('password') ?>
-    </fieldset>
-<?= $this->Form->button(__('Login')); ?>
-<?= $this->Form->end() ?>
-</div>
\ No newline at end of file
+    <div class="login-logo">
+        <b>Archer</b>APP
```
Comment (csuriel) [inline src/Template/Users/login.ctp]: Remove App

Comment (wacevedo) [inline src/Template/Users/login.ctp]: That a expect of the desing, if you see on the ticket here: https://trello.com/c/4NrceyvA/8-front-end-as-an-user-i-want-to-see-the-login-page-with-the-template-layout-implemented

Comment (csuriel) [inline src/Template/Users/login.ctp]: Remove the 'App' please...

File: src/Template/Users/login.ctp [hunk 3]
```diff
@@ -1,11 +1,44 @@
-<div class="users form">
+<?php
+use Cake\Core\Configure;
+use Cake\Error\Debugger;
+
+$this->layout = 'simple';
+
```
Comment (csuriel) [inline src/Template/Users/login.ctp]: remove this extra line

File: src/Template/Users/login.ctp [hunk 4]
```diff
@@ -1,11 +1,44 @@
...
+        </fieldset>
+
+        <div class="row">
+            <div class="col-xs-8">
+              <div class="checkbox icheck">
+                <label>
+                  <input type="checkbox"> Remember Me
+                </label>
+              </div>
+            </div>
```
Comment (csuriel) [inline src/Template/Users/login.ctp]: Add new line after HTML bloks, or delete the lines on others elements.

Comment (wacevedo) [inline src/Template/Users/login.ctp]: There  is not line on other elements now.

**Pull request merged**